plugins/store/src/store.rs

```diff
@@ -174,7 +166,57 @@ impl<R: Runtime> StoreBuilder<R> {
         self
     }
 
-    /// Builds the [`Store`].
+    /// Disable auto save on modified with a debounce duration.
+    pub fn disable_auto_save(mut self) -> Self {
+        self.auto_save = None;
+        self
+    }
+
+    /// Force create a new store even if it already exists.
+    pub fn create_new(mut self) -> Self {
+        self.create_new = true;
+        self
+    }
+
+    pub(crate) fn build_inner(mut self) -> crate::Result<(Arc<Store<R>>, ResourceId)> {
+        let stores = self.app.state::<StoreState>().stores.clone();
+        let mut stores = stores.lock().unwrap();
+        if let Some(rid) = stores.get(&self.path) {
+            return Ok((self.app.resources_table().get(*rid).unwrap(), *rid));
+        }
+
+        if stores.contains_key(&self.path) {
+            return Err(crate::Error::AlreadyExists(self.path));
+        }
+
+        let mut store_inner = StoreInner::new(
+            self.app.clone(),
+            self.path.clone(),
+            self.defaults.take(),
+            self.serialize_fn,
+            self.deserialize_fn,
+        );
+
+        if !self.create_new {
```

Comment:
Why we don't want to load the store here?

Reply:
I think because `create_new` means a new Store so don't load from disk and following modifications to the store should override the one stored in disk.

Reply:
The current description for it is `/// Force create a new store even if it already exists.`, I think a better way to do this is changing it to `returns Error on build if a store with the same path has already been loaded` and just always load the store on build

Reply:
what if you want to force create a new one and override old one?

Reply:
Then close the old one first, also the logic here doesn't do what it says, it will always return the existing store instead of creating a new one

Reply:
@lucasfernog Didn't you mean to change the behavior of `newOrExisting` (or `load` now) or is it a mistake?

Reply:
maybe we should remove that logic too, if `create_new` is true, don't return existing store

Reply:
From the description, it seems like if you try to use `create` from js side or `build` with `create_new` when there's an existing store, instead of returning an error, it will remove the old one from the resource table and put a new one in instead

This sounds good to me, it's just the descriptions and the implementations are all kinda messed up that none of them align with each other

Reply:
why call `load` if we overriding?

Reply:
Ah, I guess I misunderstood the intention here, `create_new` force creates a new empty store, I was still thinking the old behavior that it loads the store

But honestly I can't think of any use cases for this, from what I think, we can have

- Create store: new store, put it in the resource table, if the store's already there, throws (this can be used when you're sure you're only creating it once)
- Get store: get the existing store (get the store if you're sure you have created it)
- Get or create store: get or create in one lock (should be used most of the time, just give me a store, I don't care)

Reply:
Those cases are fine but creating a new store and overriding existing one can still be used as well, and we can provide an API to make it more performant instead of creating a store then calling `reset` on it. I see this as low hanging fruit that we can just add without breaking other cases.

Reply:
Yeah sure, I'll update the code and description to align the behavior, see if this is what you think it should do